poetry/masonry/builders/builder.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, poetry, env, io):
             self._path.as_posix(),
             packages=self._package.packages,
             includes=self._package.include,
+            data_files=self._package.data_files,
```

Comment:
How does this compare to 
https://github.com/sdispater/poetry/pull/539/files#diff-adffe92a131c3e1bf7e56fb29b1bc265R134